builder/ecs/client.go

```diff
@@ -117,6 +117,10 @@ const (
 	mediumRetryTimes     = 360
 )
 
+const (
+	SkipCreateImage = "you set skip_create_image true"
```

Comment:
I don't see this constant used anywhere, is this still relevant?

Reply:
Sorry, redundant constants are defined, which have been deleted in the latest commit.